src/vivarium_testing_utils/automated_validation/constants.py

```diff
@@ -24,3 +25,39 @@ def from_str(cls, source: str) -> DataSource:
 
 LOCATION_ARTIFACT_KEY = "population.location"
 POPULATION_STRUCTURE_ARTIFACT_KEY = "population.structure"
+
+
+class GBDIndexNames(NamedTuple):
```

Comment:
These aren't all GBD index names. Some of them are vivarium index names. Maybe this should be called something like `InputDataIndexNames`?